frontend/src/components/pages/RegistrantExperience/RegistrationNavStepper.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { Flex } from "@chakra-ui/react";
+import {
+  faCampground,
+  faCircleInfo,
+  faDollarSign,
+  faEdit,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import StepperTab from "../../common/StepperNavigation/StepperTab";
+import RegistrantExperienceSteps from "./RegistrationExperienceSteps";
+
+const REGISTRATION_NUM_STEPS = 4;
```

Comment:
I think we can get the num of steps by doing Object.keys(RegistrantExperienceSteps).length. Doesn't matter too much but it would make the code more maintainable